src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -450,4 +472,17 @@ private String getSSMParameter(String name) {
 		}
 	}
 
+	public String initAppVersion() {
+		Properties gitProps = loadProperties(GIT_PROPERTIES_FILENAME);
+		if (! (gitProps.containsKey(GIT_COMMIT_TIME_KEY)) && (gitProps.containsKey(GIT_COMMIT_ID_DESCRIBE_SHORT_KEY))) {
```

Comment:
this logic seems wrong to me, shouldn't there be `!` be in front of (gitProps.containsKey(GIT_COMMIT_ID_DESCRIBE_SHORT_KEY)?